dependencies.yaml

```diff
@@ -507,4 +507,4 @@ dependencies:
       - output_types: [conda, requirements, pyproject]
         packages:
           - dask-cuda==23.6.*
-          - *numba
+          - numba>=0.57
```

Comment:
Leave this `*numba` as it was, and use `&numba` above. That uses a YAML anchor to enforce that the value is copied.